packages/contract-utils/access-control/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+//! Access control module for Soroban contracts
+//!
+//! This module provides functionality to manage role-based access control in
+//! Soroban contracts.
+//!
+//! # Usage
+//!
+//! There is a single overarching admin, and the admin has enough privileges to
+//! call any function given in the [`AccessControl`] trait.
+//!
+//! This `admin` must be set in the constructor of the contract. Else, none of
```

Comment:
It might be good to mention here the mechanics of transferring the admin, that it's a critical and that's why it's a 2-step process, etc...